### ISSUE TITLE
FIX: Frontend Compilation Error

### ISSUE THREAD

**Issue by RGPosadas** [RGPosadas/LitterRally#148]
**Describe The Bug**

Docker AutoBuilds for `mull-ui` is currently failing the build due to this error:
```
ERROR in /.../Mull/apps/mull-ui/src/app/pages/register/register.tsx(42,58):
TS2322: Type '{ registrationMethod: RegistrationMethod; name: string; email: string; password: string; }' is not assignable to type 'CreateUserInput'.
  Types of property 'registrationMethod' are incompatible.
    Type 'import("/.../Mull/libs/types/src/lib/types").RegistrationMethod' is not assignable to type 'import("/.../Mull/apps/mull-ui/src/generated/graphql").RegistrationMethod'.
      Property 'LOCAL' is missing in type 'import("/.../Mull/apps/mull-ui/src/generated/graphql").RegistrationMethod'.
```
Please note that this error does not get in the way of running the frontend locally, but it does fail the build on DockerHub (see [mull-ui build](https://hub.docker.com/repository/registry-1.docker.io/ritchellegmp/mull-frontend/builds/526efb26-fff4-435f-ad72-7982d68c9d0b)).

**To Reproduce**
1. `npm run build-ui`

**Expected Behavior**

Docker AutoBuilds pass.

**Bug Handling Strategy**
- [x] Fix it now
- [ ] Fix it later

**Desktop (please complete the following information):**

 - OS: macOS Catalina 10.15.7
 - Browser: Chrome 87.0.4280.88 & Firefox 84.0.1

**Comment by cristian-aldea** [comment]
Just as a note, I noticed this and fixed it in my pr #147

**Comment by RGPosadas** [comment]
@TheGreatMarkus oh great, will close then!
